README.md

```diff
@@ -8,20 +10,26 @@ your users to easily create there own formulars.
 You will need the following things properly installed on your computer.
 
 * [Git](http://git-scm.com/)
-* [Node.js](http://nodejs.org/) (with NPM)
-* [Bower](http://bower.io/)
-* [Ember CLI](http://www.ember-cli.com/)
-* [PhantomJS](http://phantomjs.org/)
+* [nvm](https://github.com/creationix/nvm)
 
 ## Installation
 
+To make sure you are using the right version of Node.js, we are providing a `.nvmrc`. Therefore, to install all dependencies from `hopper-frontend`, make sure you have `nvm` installed.
+
 * `git clone <repository-url>` this repository
 * change into the new directory
+* `nvm install`
+* `npm install -g ember-cli`
+* `npm install -g bower`
 * `npm install`
 * `bower install`
 
+After that, you just have to type `nvm use` everytime you want to run, test or develop this project. This will make sure, that you use the correct Node.js version.
```

Comment:
Please try to avoid relative clause. It makes the text harder to read. Documentation is always easier to read if it consists of simple sentences. Remember that many readers won't be native English speakers.